deeplabcut/pose_estimation_pytorch/apis/evaluate.py

```diff
@@ -167,6 +177,196 @@ def evaluate(
     return results, predictions
 
 
+def visualize_coco_predictions(
```

Comment:
I would rename this method to `visualize_predictions`, as we should be able to visualize predictions for any type of data using it.